eo-maven-plugin/src/test/java/org/eolang/maven/AssembleMojoTest.java

```diff
@@ -150,7 +150,7 @@ void assemblesNotFailWithFailOnErrorFlag(@TempDir final Path temp) throws Except
                     )
                 )
             ),
-            Matchers.is(new Online().value())
+            Matchers.is(true)
```

Comment:
@volodya-lombrozo Why do we need this change?

Reply:
@Graur The same as [here](https://github.com/objectionary/eo/pull/2180#discussion_r1243831517).